http/src/main/scala/org/http4s/blaze/http/package.scala

```diff
@@ -27,7 +35,7 @@ package object http {
       SimpleHttpResponse("OK", 200, headers, ByteBuffer.wrap(body))
 
     def Ok(body: String, headers: Headers): SimpleHttpResponse =
-      Ok(body.getBytes(UTF_8), ("Content-Type", "text/plain; charset=UTF-8")+:headers)
+      Ok(body.getBytes(UTF_8), ("content-type", "text/plain; charset=utf-8")+:headers)
```

Comment:
http2 requires all header names to be lower case. Its [malformed otherwise](http://http2.github.io/http2-spec/index.html#HttpHeaders). I believe I make sure they are lowercase elsewhere, but why make a new string.